client/src/partials/journal/modals/journalPosting.modal.html

```diff
@@ -0,0 +1,65 @@
+<form name="JournalPosterForm" ng-submit="JournalPosterModalCtrl.submit()" bh-form-defaults novalidate>
+
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li class="static">{{ ::"TREE.POSTING_JOURNAL" | translate }}</li>
+      <li class="title">
+        {{ ::"POSTING_JOURNAL.TRIAL_BALANCE.TITLE" | translate }}
+      </li>
+    </ol>
+  </div>
+
+  <div class="modal-body">
+      <p>
+        <div class="row">
+          <div class="col-xs-4 pull-left">
+            <button type="button" ng-show="JournalPosterModalCtrl.feedBack.hasError || JournalPosterModalCtrl.feedBack.hasWarning" class="btn {{JournalPosterModalCtrl.cssClass}}" ng-click="JournalPosterModalCtrl.openErrorViewerModal()" data-method="show">
```

Comment:
Can you use `ng-class` instead?  That would reduce the need to have the function `getCssClass()` and property `cssClass`.  Those functions feel like jQuery.  The angular way to do it would be:
```html,
<button  ... ng-class="{ 'btn-error' : JournalPosterModalCtrl.feedback.hasError, 'btn-warning': JournalPosterModalCtrl.hasWarning }" ...>
  <!--- etc .. -->
</button>
```